server/util/redact/redact_test.go

```diff
@@ -109,6 +109,8 @@ func TestRedactMetadata_StructuredCommandLine(t *testing.T) {
 		{"remote_default_exec_properties", "container-registry-username=SECRET_USERNAME", "container-registry-username=<REDACTED>"},
 		{"remote_default_exec_properties", "container-registry-password=SECRET_PASSWORD", "container-registry-password=<REDACTED>"},
 		{"host_platform", "@buildbuddy_toolchain//:platform", "@buildbuddy_toolchain//:platform"},
+		{"build_metadata", "PATTERN=@//foo,NAME=@foo,PASSWORD=SECRET@bar,BAZ=", "PATTERN=@//foo,NAME=@foo,PASSWORD=bar,BAZ="},
```

Comment:
could you add a test for something like `{"build_metadata", "FOO=A=1,BAR=B=SECRET@foo", "FOO=A=1,BAR=B=@foo"}` to make sure it handles `=` inside the flag values themselves (assuming Bazel treats those as valid)

Reply:
Done. I had to change the flag parsing logic a little bit because it was matching the "A=" in the value.